src/pointer-analysis/value_set.cpp

```diff
@@ -801,8 +801,12 @@ void value_sett::get_reference_set_rec(const expr2tc &expr, object_mapt &dest)
       const irept &a = sym->type.find("alignment");
       if (a.is_not_nil())
       {
-        assert(a.is_constant());
```

Comment:
I debugged this line of the code and printed the expr for `a`.
```
shl
  * type: signedbv
      * width: 32
      * #cpp_type: signed_int
  * operands: 
    0: constant
        * type: signedbv
            * width: 32
            * #cpp_type: signed_int
        * value: 00000000000000000000000000000001
        * #location: 
          * file: main3.c
          * line: 9
          * function: main
          * column: 34
        * #cformat: 1
    1: constant
        * type: signedbv
            * width: 32
            * #cpp_type: signed_int
        * value: 00000000000000000000000000000111
        * #location: 
          * file: main3.c
          * line: 9
          * function: main
          * column: 34
        * #cformat: 7
  * #location: 
    * file: main3.c
    * line: 9
    * function: main
    * column: 34
```
This is actually still constant, but it's a computation, maybe because the assertion is too strict?